app/api/users.py

```
@@ -0,0 +1,67 @@
+from app.services import user_service
+
+router = APIRouter(prefix="/api/users", tags=["users"])
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/api/users/login")
+
+
+@router.post(
+    "/register",
+    response_model=UserResponse,
+    status_code=status.HTTP_201_CREATED,
+)
+async def register_user(user_in: UserCreate, db: AsyncSession = Depends(get_db)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)